### PR TITLE
Add informations on code signed downloads for Windows and macOS

### LINKED ISSUE #131: Mention how to bypass macOS security restrictions on the macOS download page
Right now, the [macOS download page](https://godotengine.org/download/osx) doesn't mention how to bypass Gatekeeper to run the binary as it's unsigned. The process apparently got more complex in macOS 10.15, so the instructions should take this into account.

See https://github.com/godotengine/godot/issues/11984.

### PR DESCRIPTION
**Pull request by Calinou**
See https://github.com/godotengine/godot/issues/11984.

This closes #131.

### CODE REVIEW

File: themes/godotengine/pages/download/osx.htm [hunk 1]
```diff
@@ -26,7 +26,16 @@
 {% endput %}
 
 {% put instructions %}
-  <p>You can also get it with <a href="https://brew.sh/">Homebrew</a>.</p>
+  <p>
+    Godot is currently not code-signed for macOS. See the last section of
+    <a href="https://support.apple.com/en-us/HT202491">this page</a>
+    for instructions on allowing Godot to run anyway. Alternatively, you can install
+    <a href="http://store.steampowered.com/app/404790">Godot from Steam</a> to work around this.
```
Comment (mhilbrunner) [inline themes/godotengine/pages/download/osx.htm]:
```suggestion
    <a href="https://store.steampowered.com/app/404790">Godot from Steam</a> to work around this.
```

File: themes/godotengine/pages/download/windows.htm [hunk 2]
```diff
@@ -37,11 +37,12 @@
 {% endput %}
 
 {% put instructions %}
-  <p>You can also get it with <a href="http://scoop.sh/">Scoop</a></p>
+  <p>You can also get Godot with <a href="http://scoop.sh/">Scoop</a>.</p>
```
Comment (mhilbrunner) [inline themes/godotengine/pages/download/windows.htm]:
```suggestion
  <p>You can also get Godot with <a href="https://scoop.sh/">Scoop</a>.</p>
```

**Review by mhilbrunner — COMMENTED** [review]
Two small things, otherwise looks good to me. :)

**Comment by Calinou** [comment]
@akien-mga Is this good to merge now?

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**